lib/fluent/supervisor.rb

```diff
@@ -79,9 +82,15 @@ def after_run
       stop_windows_event_thread if Fluent.windows?
       stop_rpc_server if @rpc_endpoint
       stop_counter_server if @counter
+      cleanup_lockdir
       Fluent::Supervisor.cleanup_resources
     end
 
+    def cleanup_lockdir
+      FileUtils.rm(Dir.glob(File.join(@fluentd_lockdir, "fluentd-*.lock")))
+      FileUtils.rmdir(@fluentd_lockdir)
```

Comment:
```suggestion
      FileUtils.rmdir(@fluentd_lockdir)
    rescue StandardError
      # do nothing
```
Should we rescue exceptions for subsequent processing?

Reply:
We can use `FileUtils.rm_rf(@fluentd_lockdir)` but we don't need to use this method if we use `Dir.mktmpdir do ... end`. `Dir.mktmpdir do ... end` removes the temporary directory automatically.

Reply:
Oh, I'm sorry. We should use the block of `mktmpdir` and we don't need this change.

Reply:
Thank you! I fixed it in 75ef92f787106de0a054174c2423a7f2fee00350, in such a way that we don't need to do any manual clean up any more.